google/ggrc-core
# GGRC-4756 Update propagation tree for program editor

# Issue description

Program Editor can delete objects mapped to program

# Steps to test the changes

1. Create a Program
2. Assign Global Creator as Program Editor for the Program
3. Create a new Issue in scope of the Program
4. Log as Global Creator and open Program page Issues tab
5. Try to delete Issue
6. See that you can not delete Issue

# Solution description

Update propagation tree for program editor

# Sanity checklist


# Migration checklist

# PR Review checklist
